integration-test/helpers/harness.js

```diff
@@ -41,7 +41,7 @@ export async function withEmailProtectionExtensionSignedInAs (page, username) {
  *     MAX_INPUTS_PER_PAGE: number,
  *     MAX_FORMS_PER_PAGE: number,
  *     MAX_INPUTS_PER_FORM: number,
- *     MAX_FORM_MUT_OBS_COUNT: number
+ *     MAX_FORM_RESCANS: number
```

Comment:
Renamed to capture the fact that the limit is per scan, regardless of whether the mutation observer initiated the rescan. This was at the heart of the issue we were seeing, where the form could rescan the same forms over and over without running into a failsafe.